conformance/utils/http/http.go

```diff
@@ -144,40 +142,31 @@ func awaitConvergence(t *testing.T, threshold int, fn func() bool) {
 	}
 }
 
-// WaitForConsistency repeats the provided request until it completes with a response having
+// ExpectResponseEventually repeats the provided request until it completes with a response having
 // the expected status code consistently. The provided threshold determines how many times in
 // a row this must occur to be considered "consistent".
```

Comment:
Nit: Maybe "WaitForConsistentResponse"? Without that in the function name, the "consistent" terminology in this comment is not very meaningful.

Reply:
done, thanks